packages/choice-input/src/ChoiceGroupMixin.js

```diff
@@ -119,24 +143,25 @@ export const ChoiceGroupMixin = dedupeMixin(
       }
 
       _getCheckedElements() {
-        const filtered = this.formElementsArray.filter(el => el.checked === true);
-        if (this.multipleChoice) {
-          return filtered;
-        }
-        return filtered.length > 0 ? filtered[0] : undefined;
+        // We want to filter out disabled values out by default
+        return this.formElements.filter(el => el.checked && !el.disabled);
+        // if (this.multipleChoice) {
```

Comment:
why leave commented out code in there? imho should be removed

Reply:
Definitely :) Alex pointed this already out as well. I'm on it